modules/search/module.py

```
@@ -487,45 +539,105 @@ def perform_search(self):
+        """Query the SQLite index and populate results immediately.
+
+        Returns True if results were found and displayed, False if the caller
+        should fall back to a filesystem search.
+        """
+        try:
+            results = self._index.search_jobs(
+                term, search_customer, search_job, search_desc, search_drawing,
+            )
+            if include_blueprints:
+                results += self._index.search_bp(term)
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)